From c7ba03fdaaffaeb0d520ceb668a85ab5712bdbad Mon Sep 17 00:00:00 2001
From: Chaitanya Kulkarni <kch@nvidia.com>
Date: Mon, 22 May 2023 03:59:52 -0700
Subject: [PATCH 1/4] nvme-fabrics: fix qid in error message

When secure concatenation is not implemented, instead of statically
printing the qid=0 in warning message print the qid received from caller
as for I/O queues qid can be non zero.

Signed-off-by: Chaitanya Kulkarni <kch@nvidia.com>
---
 drivers/nvme/host/fabrics.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/nvme/host/fabrics.c b/drivers/nvme/host/fabrics.c
index b1fa27b60917..529a86aea5f5 100644
--- a/drivers/nvme/host/fabrics.c
+++ b/drivers/nvme/host/fabrics.c
@@ -554,7 +554,8 @@ int nvmf_connect_io_queue(struct nvme_ctrl *ctrl, u16 qid)
 		/* Secure concatenation is not implemented */
 		if (result & NVME_CONNECT_AUTHREQ_ASCR) {
 			dev_warn(ctrl->device,
-				 "qid 0: secure concatenation is not supported\n");
+				 "qid %d: secure concatenation is not supported\n",
+				 qid);
 			ret = NVME_SC_AUTH_REQUIRED;
 			goto out_free_data;
 		}
-- 
2.40.0

